build/Dockerfile

```diff
@@ -13,20 +11,17 @@ LABEL 	io.k8s.display-name="OpenShift PSAP Ci artifacts" \
 COPY build/root /
 
 # Install openshift-ansible RPMs and some debugging tools
-RUN yum install -y epel-release \
-    && yum update -y \
-    && yum install --setopt=tsflags=nodocs -y \
-      jq wget git make vim \
-      inotify-tools ansible \
-      python3 openssh-clients \
- && yum clean all \
- && rm -rf /var/cache/yum
-
-# Get kubectl binary
-ENV OC_RELEASE=4.5.14
-RUN wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OC_RELEASE/openshift-client-linux-$OC_RELEASE.tar.gz \ 
-	&& tar -C /usr/local/bin -xzf openshift-client-linux-$OC_RELEASE.tar.gz \
-	&& rm -f openshift-client-linux-$OC_RELEASE.tar.gz
+RUN yum install -y \
+        glibc-langpack-en \
+        git make jq vim wget \
+        python3 python3-devel python3-pip python3-setuptools && \
+        pip3 install --no-cache-dir --upgrade setuptools pip && \
+        pip3 install --no-cache-dir \
+        openshift \
+        ansible==2.9.* && \
+        yum clean all && \
+        rm -rf $HOME/.cache && \
+        rm -rf /var/cache/yum
```

Comment:
I think the indentation got lost since a previously version?

Reply:
since I moved the `&&` to the line before, now looks like this, but I get what you mean
fixing

Reply:
Done